src/gallery.tsx

```diff
@@ -42,10 +55,7 @@ export interface GalleryProps {
   /**
    * Triggers before PhotoSwipe.init() call
    *
-   * Use it for something that you need to do before PhotoSwipe.init() call -
-   * for example, you can use it for adding custom UI elements
-   *
-   * https://photoswipe.com/adding-ui-elements
+   * Use it for something that you need to do before PhotoSwipe.init() call
```

Comment:
Maybe use the same description as for the `onOpen` hook?

```suggestion
   * Use it for accessing PhotoSwipe API
```

and so on